spring-integration-xmpp/src/main/java/org/springframework/integration/xmpp/outbound/ChatMessageSendingMessageHandler.java

```diff
@@ -40,8 +47,11 @@
  */
 public class ChatMessageSendingMessageHandler extends AbstractXmppConnectionAwareMessageHandler {
 
+	private static final Pattern xmlPattern = Pattern.compile("<(\\S.*?)>(.*?)</\\1>");
```

Comment:
Is it possible the tag might have attributes? If so, this won't work, we'd need something like

`<(\\S[^>\\s]*)[^>]*>`